src/audio/disk_noise.cpp

```diff
@@ -532,17 +554,23 @@ static void init_disknoise_dosbox_settings(SectionProp& secprop)
 {
 	constexpr auto OnlyAtStart = Property::Changeable::OnlyAtStart;
 
-	auto* bool_prop = secprop.AddBool("hard_disk_noise", OnlyAtStart, false);
-	bool_prop->SetHelp(
+	auto* str_prop = secprop.AddString("hard_disk_noise", OnlyAtStart, "off");
```

Comment:
Instead of `OnlyAtStart`, can you change this to `Always`? That will cause the entire disk noise module to be destructed then re-inited, which is fine (we don't need to overboard with optimisations like not reloading the samples; it's fast enough, so it's pointless).

Also in the future, users will have the option to toggle disk noise sounds from the GUI (OSD).